package.json

```diff
@@ -15,7 +15,11 @@
     "codegen": "turbo run codegen",
     "prettier": "prettier --check \"**/*.{ts,tsx,md}\"",
     "prettier:fix": "prettier --write \"**/*.{ts,tsx,md}\"",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "crowdin": "crowdin",
```

Comment:
do we need this?

Reply:
No, not really, it's not essential for anything to work. It came from here: https://lingui.js.org/tools/crowdin.html#add-the-following-scripts
I guess it's just a handy helper to be able to write "yarn crowdin" followed by any crowdin parameters.